cmd/fleet/main.go

```diff
@@ -511,6 +511,7 @@ LOOP:
 		}
 
 		curCfg = newCfg
+		f.cfg = curCfg
```

Comment:
doesn't look like curCfg is used after this point

why not just do
```
f.cfg = newCfg
```

and remove 
```
curCfg = newCfg
```

Reply:
it is used in the for loop for checking to see if the configuration change required a reload

Reply:
ughh, didn't notice the bigger loop it's in ... long function